src/main/java/goral/psychotherapistoffice/domain/messeges/MessageService.java

```diff
@@ -0,0 +1,56 @@
+package goral.psychotherapistoffice.domain.messeges;
+
+import goral.psychotherapistoffice.domain.messeges.dto.MessageDto;
+import jakarta.mail.*;
+import jakarta.mail.internet.InternetAddress;
+import jakarta.mail.internet.MimeMessage;
+import org.springframework.stereotype.Service;
+
+import java.util.Properties;
+
+
+@Service
+public class MessageService{
+
+    public boolean sendMail(MessageDto messageDto) {
+        boolean flag = false;
+
+        //logic
+        //smtp properties
+        Properties properties = new Properties();
+        properties.put("mail.smtp.auth", true);
+        properties.put("mail.smtp.starttls.enable", true);
+        properties.put("mail.smtp.port", "587");
+        properties.put("mail.smtp.host", "smtp.gmail.com");
+
+        final String username = "tomasz.gorski88@gmail.com";
```

Comment:
in php usually we are creating hide .env file and we can move this values to file

https://www.geeksforgeeks.org/environment-variables-in-java/